components/organisms/Footer.js

```diff
@@ -0,0 +1,99 @@
+import { format } from "prettier";
+import PropTypes from "prop-types";
+import { ActionButton } from "../atoms/ActionButton";
+import Image from "next/image";
+
+/**
+ * footer element for all pages
+ */
+
+export function Footer(props) {
+  return (
+    <div className="w-full mt-4">
+      <div className="w-full h-48 flex flex-col md:flex-row  bg-footer-blue bg-footer-parliament-image bg-no-repeat bg-right-bottom text-white">
+        <div className="w-full flex flex-col mt-2 mb-2 layout-container pt-4">
+          <div className="w-full flex ">
+            <div className="ml-2 md:ml-0 mr-2 md:mr-0 flex md:items-start mb-2">
+              <p className="mb-4 mt-2 text-sm">
+                {props.reportDescription}
+                <br></br> {props.reportDescriptionContinue}
+              </p>
```

Comment:
So it doesn't matter where the linebreaks are. Once you have a responsive website, you can't really control for linebreaks anymore. 

More importantly though, this part of the footer is being redone in the latest mocks. Now we have a navy blue background with a bunch of links in it, and the report a problem button is above: https://www.figma.com/file/t5uDkPa86nsS3ssFESX2Em/BDM-DECD-Alpha-Site-Sprint-2?node-id=105%3A3801